tools/debugging/json-log-to-html.py

```diff
@@ -0,0 +1,202 @@
+"""
+Utility to format Raiden json logs into HTML.
+Colorizes log key-values according to their hash to make debugging easier.
+Allows to filter records by `event`.
+When processing multiple files make sure to set PYTHONHASHSEED to a fixed value,
+otherwise the colors of values will not match up.
+"""
+
+import hashlib
+import json
+from collections import Counter, namedtuple
+from datetime import datetime
+from html import escape
+from json import JSONDecodeError
+
+import click
+
+try:
+    from colour import Color
+except ImportError:
+    click.secho('This tool requires the `colour` package.')
```

Comment:
Why not also add it to the dev-requirements?

Reply:
No reason really, I'll add it.

Reply:
Done